ecommerce_integrations/shopify/page/shopify_import_products/shopify_import_products.js

```diff
@@ -33,9 +33,7 @@ shopify.ProductImporter = class {
 	}
 
 	async checkSyncStatus() {
-
-		const { message: jobs } = await frappe.db.get_list("RQ Job", {filters: {"status": ("in", ("queued", "started"))}});
-
+		const jobs = await frappe.db.get_list("RQ Job", {filters: {"status": ("in", ("queued", "started"))}});
```

Comment:
Is there any way we can run version-specific code from js? Because, this doesn't work in version-13 as it doesn't have RQ Jobs.

Reply:
We can check version from frappe.boot.versions (I don't remember actual key, but it should be there)

Reply:
Yes. Got it. 

```js
parseInt(frappe.boot.versions.frappe); // returns 13, 14, or 15
```

Which is better? Calling separate functions based on the frappe version or having separate branches of this repo like `version-13` and `version-15`?

Or is `version-13` outdated enough to warrant an upgrade?

Reply:
V13 will be EOL in 2 months. Maintaining multiple branches is kind of a hassle.

Reply:
Diff code based on condition is fine.

Reply:
Understood. But again, we only have one customer on v13 using this. I'm not sure if there are any others still on v13 with this app. 

I could just fork this and keep the old code on it instead of changing the upstream for one single customer.

It will also prevent any future updates messing with v13.